src/include/stir/RadionuclideDB.h

```diff
@@ -73,16 +73,16 @@
 
 START_NAMESPACE_STIR
 
-class RadionuclideDBProcessor//: public Radionuclide
+class RadionuclideDB//: public Radionuclide
```

Comment:
```suggestion
class RadionuclideDB
```
Also, the doxygen needs to be just before the class (i.e. move `START_NAMESPACE_STIR` up)